unyt/tests/test_unyt_testing.py

```diff
@@ -40,3 +42,67 @@ def test_atol_conversion_error():
     a2 = unyt_array([1.0, 2.0, 3.0], "cm")
     with pytest.raises(AssertionError):
         assert_allclose_units(a1, a2, atol=unyt_quantity(0.0, "kg"))
+
+
+def test_accepts():
+    @accepts(a=time, v=length / time)
+    def foo(a, v):
+        return a * v
+
+    _ = foo(a=2 * second, v=3 * meter / second)
```

Comment:
If we're not going to use the return value, might as well not capture it (this comment applies to a handful more lines)
```suggestion
    foo(a=2 * second, v=3 * meter / second)
```